javadoc/reference/com/esri/android/geotrigger/GeotriggerBroadcastReceiver.ProcessLocationFailListener.html

```diff
@@ -0,0 +1,501 @@
+<!DOCTYPE html>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
result of the javadoc generation, you will see that in all the generated html files.